python/test/unit/language/test_core.py

```diff
@@ -3039,7 +3045,8 @@ def kernel(X, stride_xm, stride_xk, Y, stride_yk, stride_yn, W, stride_wn, strid
             z = num / den[:, None]
         if CHAIN_DOT:
             w = tl.load(Ws)
-            z = tl.dot(z.to(w.dtype), w, input_precision=INPUT_PRECISION, out_dtype=out_dtype)
+            t = z.to(w.dtype)
+            z = tl.dot(t, w, input_precision=INPUT_PRECISION, out_dtype=out_dtype)
```

Comment:
is this just a clean up change or does this affect functionality of the interpreter?

Reply:
Yeah, I'll take out debugging info...

Reply:
This was left just to debug t's value